inc/class-wpseo-image-utils.php

```diff
@@ -371,10 +371,11 @@ private static function has_usable_dimensions( $dimensions, $usable_dimensions )
 	 *
 	 * @return string|null The image URL.
 	 */
-	public static function get_first_usable_content_image_for_post( $post_id = null ) {
+	public static function get_first_usable_content_image_for_post( $post_id ) {
```

Comment:
Changing this will break the situation where no argument is given. I would love to see this reverted.

Reply:
Hi @andizer! Hm but is there any case where we're not passing a post ID? I'd tend to think an ID should always be passed. I guess same applies to `get_first_content_image_for_term( $term_id )`

Reply:
For us there isn't a situation, but since this method is public it might be possible other plugins or themes are using the method. That the reason that I should leave this method as it was. 

For the `get_first_content_image_for_term` we can force the `$term_id`, because we are introducing this method in this pull request.